editor.planx.uk/src/components/EditorNavMenu/EditorNavMenu.tsx

```diff
@@ -123,9 +125,16 @@ function EditorNavMenu() {
       route: `/${teamSlug}/submissions`,
       accessibleBy: ["platformAdmin", "teamEditor", "demoUser"],
     },
+    {
+      title: referenceCode ? `Planning Data (external link)` : `Planning Data unavailable`,
+      Icon: LayersIcon,
+      route: referenceCode ? `https://submit.planning.data.gov.uk/organisations/local-authority:${referenceCode}` : `#`,
+      accessibleBy: "*",
```

Comment:
Planning Data is a public open data platform, so making these links visible to all in editor - does that seem right?